impl/src/main/java/org/jboss/weld/resolution/ResolvableBuilder.java

```diff
@@ -264,10 +265,13 @@ public String toString() {
 
         @Override
         public int hashCode() {
-            int result = 17;
```

Comment:
Did you run into a code path where hashCode is called more than once for a given resolvable? Unless I am wrong it should be called exactly once most of the time.